calico-vpp-agent/services/service.go

```diff
@@ -31,6 +33,16 @@ const (
 	hashConfigReverse   hashConfig = "reverse"
 )
 
+var hashConfigMap = map[hashConfig]types.IPFlowHash{
+	hashConfigDstaddr:   types.FlowHashDstIP,
+	hashConfigDstport:   types.FlowHashDstPort,
+	hashConfigIproto:    types.FlowHashProto,
+	hashConfigReverse:   types.FlowHashReverse,
+	hashConfigSrcaddr:   types.FlowHashSrcIP,
+	hashConfigSrcport:   types.FlowHashSrcPort,
+	hashConfigSymmetric: types.FlowHashSymetric,
+}
+
```

Comment:
Instead of doing an extra layer of mangling here, we could directly build the bitmap in `ParseServiceAnnotations()` as we already are doing a switch statement there